README.md

```diff
@@ -96,6 +96,12 @@ Alternatively, if you already have s sSMTP config, you can use that config with:
 
 See the following link for details on setting up sSMTP: [SSMTP - ArchWiki](https://wiki.archlinux.org/index.php/SSMTP)
 
+# Adding custom alarms, charts and configuration overrides
+
+To add custom alarms, charts or to override any default configuration file, mount a volume to the container to /etc/netdata/overrides, like `-v /opt/netdata/overrides:/etc/netdata/overrides`.  Then, place your config files in the directory as if it was /etc/netdata/.  
```

Comment:
typo in the README: it's `/etc/netdata/override` without a trailing `s`.

Reply:
oops.  Thanks for catching that!

Reply:
https://github.com/titpetric/netdata/pull/56 should fix this